test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,103 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+cleanup=0
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
+
+	exit 1
+}
+
+
+while :; do
+	case "$1" in
+	-j)
+		parallel_jobs="$2"
+		[[ "$parallel_jobs" =~ ^[1-9][0-9]*$ ]] ||
+			fatal_msg "-j option argument must be a non-zero integer"
+		shift ; shift ;;
+	-*)
+		usage ;;
+	*)
+		break ;;
+	esac
+done
+
+# Wait for other format runs.
+wait_for_process()
+{
+	while true
+	do
+		for process in ${PID_LIST[@]};do
+			if ps $process > /dev/null ; then
```

Comment:
From a consistency point of view, you use a blend of conditions, sometimes with the old style and sometimes with the newer bash [[ XXX ]] style. It would be easier to read if it was consistent.

Reply:
Thanks for pointing this out. Fixed in cffdd7b675407ff0cbd858e66d7db24330bc6541